src/main/java/mate/academy/service/UserService.java

```diff
@@ -8,13 +8,22 @@ public class UserService {
             new User("alice@i.ua", "1234")
     };
 
+    public static User[] getUsers() {
+        return users;
+    }
```

Comment:
as a Java programmer, your job is to implement only that features that mentioned in the task
![image](https://github.com/mate-academy/jv-oop-lesson-0/assets/73041091/f2554b88-e441-4272-a9ac-39caf28ee711)

```suggestion
```